src/runtime/test/interface-info-test.ts

```diff
@@ -202,7 +202,7 @@ describe('interface', () => {
     // Initially handle type are unresolvable type variables.
     assert.lengthOf(recipe.handles, 2);
     for (const handle of recipe.handles) {
-      const collectionType = (handle.type as CollectionType).collectionType;
+      const collectionType = (handle.type as CollectionType<Type>).collectionType;
```

Comment:
Can you try just removing the 'as CollectionType<Type>' syntax?  The generics should be working in this case.

Reply:
VScode doesn't seem to like it and marks `handle.type` as type `Type`. From my budding understanding of generics, the cast to `CollectionType<Type>` is necessary b/c `handle.type` could be of any `Type`. 

However, once it is cast to `Collection<Type>` and we obtain the underlying `collectionType`, the `collectionType` no longer needs to be cast. For example, in the case of `test/recipes/particle-tests.ts`, once `listConnType` is cast to a `Collection<TypeVariable>`, `listConnType.collectionType` is automatically assumed to be of type `TypeVariable` and would no longer need to be cast thanks to the generics.